gm4_metallurgy/data/gm4_metallurgy/function/smooshing/remove_band/clear_data.mcfunction

```diff
@@ -2,12 +2,30 @@
 # run from smooshing/remove_band/check
 
 execute if score custom_model_data gm4_ml_data matches 0 run data remove entity @s Item.components."minecraft:custom_model_data"
+execute if items entity @s contents *[custom_data~{gm4_metallurgy:{active_shamir:"infinitas"}}] run data remove entity @s Item.components."minecraft:max_stack_size"
```

Comment:
Why does infinitas need special handling? Maybe include a comment about this in the function